pandas/tests/extension/test_boolean.py

```diff
@@ -139,7 +140,7 @@ def _check_op(self, s, op, other, op_name, exc=NotImplementedError):
                 expected[result.isna()] = np.nan
             self.assert_series_equal(result, expected)
         else:
-            with pytest.raises(exc):
+            with pytest.raises(exc, match=msg):
```

Comment:
Is this actually getting hit? Might be misreading but seems like `msg` wouldn't be defined in this block

Reply:
Thanks @WillAyd. You are right, msg is not getting hit. I've removed it.